plugins/rapid7_insightvm_cloud/unit_test/test_vuln_search.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import os
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from icon_rapid7_insightvm_cloud.actions.vuln_search import VulnSearch
+from icon_rapid7_insightvm_cloud.actions.vuln_search.schema import Input
+from icon_rapid7_insightvm_cloud.connection.schema import Input as ConnectionInput
+from unittest.mock import patch
+from unit_test.utils import Utils
+from unit_test.mock import (
+    mock_request,
+)
+
+
+class TestVulnSearch(TestCase):
+    @classmethod
+    def setUpClass(self) -> None:
+        self.params = {
+            "asset_criteria": "last_scan_end > 2000-01-01T00:00:00.000Z",
+            "asset_criteria_invalid": "invalid asset criteria",
+            "size": 10,
+            "sort_criteria": {"risk-score": "asc", "criticality-tag": "desc"},
+            "vuln_criteria": "severity IN ['Critical', 'Severe']",
+            "vuln_criteria_invalid": "invalid vuln criteria",
+        }
+
+    def setUp(self) -> None:
+        self.connection, self.action = Utils.default_connector(VulnSearch())
+
+    # test finding event via all inputs
+    @patch("requests.request", side_effect=mock_request)
+    def test_vuln_search_all_inputs(self, _mock_req):
+        actual = self.action.run(
+            {
+                Input.ASSET_CRITERIA: self.params.get("asset_criteria"),
+                Input.SIZE: self.params.get("size"),
+                Input.SORT_CRITERIA: self.params.get("sort_criteria"),
+                Input.VULN_CRITERIA: self.params.get("vuln_criteria"),
+            }
+        )
+        expected = Utils.read_file_to_dict("expected_responses/vuln_search.json.resp")
+        self.assertEqual(expected, actual)
+
+    # test finding event via all inputs
```

Comment:
I think the test comment is repeated throughout, would you be able to update this please?